### PR TITLE
darecode.md

### PR DESCRIPTION
**Pull request by manuelbcd**
This is a modified version of the [Contributing Guidelines](https://github.com/remoteintech/remote-jobs/blob/master/CONTRIBUTING.md).

This pull request adheres to the repository's [Code of Conduct](https://github.com/remoteintech/remote-jobs/blob/master/CODE_OF_CONDUCT.md).

- [x] I am an employee of the company mentioned and confirm all included details are correct
- [ ] This PR contains housekeeping only (URL edits, copy changes etc)
- [x] You know your alphabet - company is listed in alphabetical order in the README
- [x] The company directly hires employees. No bootcamps / freelance sites / etc
- [x] The company hires remote employees, or positions are available to remote workers and are clearly illustrated as such
- [x] A [company profile](https://github.com/remoteintech/remote-jobs/blob/master/company-profiles/example.md) is included - __Required__ for new additions. (This can be a basic outline but at least something please)
- [x] __Remote status__ has details regarding how the culture includes remote employees, how the company integrated remote workers, etc
- [x] __Region__ details the geographic regions in which this company's employees can reside. For more details see the instructions in the [example company profile](/company-profiles/example.md#region).
- [x] __How to apply__ details the best approach for new applications, page on site where open position are listed, and any other help available for job hunters

### CODE REVIEW

**Comment by auto-nylen** [comment]
Hi @manuelbcd, thanks for your contribution!

Our automated checks detected some problems with this pull request (as of commit f792741).  See below for details:

```
darecode.md: No link to company profile from readme

1 problem detected
```

Please fix these issues and push a new commit to this pull request.

**Comment by manuelbcd** [comment]
Link added

**Comment by nylen** [comment]
Hi @manuelbcd, you need to edit the `README.md` file to include a link to this new company profile, and then we can merge this.

File: company-profiles/darecode.md [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+
+We’re 50 ppl in 2019 and aim to growth safely to reach 75 team members in y2020.
+
+## Remote status
+
+We have offices in Spain (Madrid and Zaragoza) and several remote professionals in several cities. We work with local and remote members.
+
+## Region
+
+Worldwide but specially European market. Our customers are in several locations.
```
Comment (nylen) [inline company-profiles/darecode.md]: Typo: "specially" &rarr; "especially" (this is one of the few words that start with "es" in both English and Spanish!)

Comment (manuelbcd) [reply company-profiles/darecode.md]: Thank you @nylen! you are right

Fixed

Best regards

**Comment by manuelbcd** [comment]
> Hi @manuelbcd, you need to edit the `README.md` file to include a link to this new company profile, and then we can merge this.

HI @nylen thanks for the feedback. I'm not sure what are you referring to. The first line of Readme includes a link to our site. Isn't it right?

**Comment by nylen** [comment]
No, the main readme file for the project, that has the table that links to all the companies: https://github.com/remoteintech/remote-jobs/blob/master/README.md

**Comment by auto-nylen** [comment]
Hi @manuelbcd, thanks for updating the pull request!  According to our automated checks, everything looks OK now (as of commit 9c92049). :heavy_check_mark:

**Comment by manuelbcd** [comment]
Nice, thank you.  Company has been added right now.

**Comment by nylen** [comment]
Thanks! Merging this now.

**Pull request merged**